linera-sdk/src/test/validator.rs

```diff
@@ -195,6 +195,11 @@ impl TestValidator {
         self.new_chain_with_keypair(key_pair).await
     }
 
+    /// Add an existing [`ActiveChain`]
```

Comment:
Nit: changing the verb conjugation to follow the convention and adding a period at the end.

```suggestion
    /// Adds an existing [`ActiveChain`].
```